### ISSUE TITLE
Some UI enhancements (SimpleX 6.0-3)

### ISSUE THREAD

**Issue by serrq** [simplex-chat/simplex-chat#4609]
In the first screen, I show the right places (yellow 'x') because my display is much wide and I cannot reach comfortably the highlighted button.

In the second screen, the job is great, but I would like an option to move pinned chats above or bottom, as my needs. Currently it allows only at the bottom.

PS: private notes are not always a priority. So they can be moved above, but I would having the possibility to change my mind (bottom).

PS2: adding an option called "more close buttons"? y/n

![1000116143](https://github.com/user-attachments/assets/69906b02-b048-4d52-8d8a-1d1d86e3320c)

--------------

![1000116145](https://github.com/user-attachments/assets/a8bdc81c-883d-49d7-9845-47b2c32787d8)

**Comment by greensheeps** [comment]
> Some buttons yet remain uncomfortable to reach with fingers (6.7" display).
> 
> ![1000116149](https://private-user-images.githubusercontent.com/113377431/355687938-6755cad2-8c9f-4acd-a114-b8b20d4873a1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxMzM2MTUsIm5iZiI6MTcyMzEzMzMxNSwicGF0aCI6Ii8xMTMzNzc0MzEvMzU1Njg3OTM4LTY3NTVjYWQyLThjOWYtNGFjZC1hMTE0LWI4YjIwZDQ4NzNhMS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwOFQxNjA4MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYmVlNjE4NTlkZWEzMjA1NTIxNzRjNGRlZjE5ODBhNTdlNmE5YTY5Y2U0MWRjMDdjN2VjZWJiOTJlNzUyYTFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._S9NsPOQGsrR6_-F9xgSy7cfZR_A8ViLju5HCuIWBPw)

Is this really an issue since you have the system back button at the bottom?

**Comment by greensheeps** [comment]
> > Is this really an issue since you have the system back button at the bottom?
> 
> I don't use the hardware buttons when I have the software ones. However, if the dev team doesn't want to move the buttons closer, no problem.
> 
> PS: my phone allows also "no buttons" UI

Your phone has hardware buttons for going back? I was taking about the software 3-button navigation visible in your screenshot.

**Comment by greensheeps** [comment]
> > the software 3-button navigation visible in your screenshot.
> 
> You see these buttons because I hardly chosen the UI with 3-buttons at the bottom. It is optional.
> 
> In my smartphone there is another UI option without 3-buttons, with just swipe actions.
> 
> Your observation could be right if, only if, I choose the 3-buttons UI (that it is my case) but at this point, that button would be useless up there.
> 
> Who would going in that up corner to make an uncomfortable tap?

Right, but the other option on most phones is gesture controls where you can do the back gesture easily from any part of the screen. You mentioned hardware buttons which aren't common anymore so that's all I was asking about.
